README.md

```diff
@@ -86,7 +86,11 @@ The version format for this repo is `{major}.{minor}.{patch}` for stable, and
 `{major}.{minor}.{patch}-{stage}.{devnum}` for unstable (`stage` can be alpha or beta).
 
 To issue the next version in line, specify which part to bump,
-like `make release bump=minor` or `make release bump=devnum`.
+like `make release bump=minor` or `make release bump=devnum`. This is typically done from the
+master branch, except when releasing a beta (in which case the beta is released from master,
+and the previous stable branch is released from said branch). To include changes made with each
+release, update "docs/releases.rst" with the changes, and apply commit directly to master 
```

Comment:
Nothing about this PR but it reminds of the fact that we haven't yet merged the `towncrier` changes upstream: https://github.com/ethereum/ethereum-python-project-template/pull/28

We were holding it back to iron out issues in the release script.

By now we can probably sync it with the latest release script from Trinity and then get that in.

Reply:
Yeah, if the next trinity release is smooth with towncrier, I'm game to merge it to the template!